### PR TITLE
Issue #11214: specified violation message in WhitespaceAroundCheck

### PR DESCRIPTION
**Pull request by Kevin222004**
Issue #11214: specified violation message in WhitespaceAroundCheck

### CODE REVIEW

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/whitespacearound/InputWhitespaceAroundAllowEmptyTypesAndNonEmptyClasses2.java [hunk 1]
```diff
@@ -23,29 +23,30 @@
 import java.util.function.Function;
 import java.util.function.Supplier;
 
-public class InputWhitespaceAroundAllowEmptyTypesAndNonEmptyClasses2{ // violation
+public class
+    InputWhitespaceAroundAllowEmptyTypesAndNonEmptyClasses2{ // violation ''{' is not .* whitespace'
```
Comment (romani) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/whitespacearound/InputWhitespaceAroundAllowEmptyTypesAndNonEmptyClasses2.java:27]: We need more space for message, it is not clear what message is.
Please make this line without violation and create new top level class ( not public) in this file with violation.

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/whitespacearound/InputWhitespaceAroundAllowEmptyTypesAndNonEmptyClasses2.java [hunk 2]
```diff
@@ -23,29 +23,30 @@
...
 
-    class TestClass { int a; }int b; // violation
+    class TestClass { int a; }int b; // violation ''}' is not followed by whitespace'
 
     class Table {}
 
     interface SupplierFunction<T> extends Function<Supplier<T>, T> {}
 
-    class NotEmptyClass{ public void foo1() { foo2(); } } // violation
+    class NotEmptyClass{ public void foo1() { foo2(); } } // violation ''{' is not .* whitespace'
```
Comment (romani) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/whitespacearound/InputWhitespaceAroundAllowEmptyTypesAndNonEmptyClasses2.java:49]: Shorten name of class, remove not required visibility modifiers, to give message whole space

**Review by romani — CHANGES_REQUESTED** [review]
Same all similar cases:

**Comment by Kevin222004** [comment]
@Vyom-Yadav  @romani can you please help me with this, 
Screenshot from 2022-03-06 12-41-11

I have tried to make  changes in  WhitespaceAroundCheckTest.java  but it wont works

**Comment by romani** [comment]
Extra violation in line 26. Please sync amount of violation lines in input file and in test method in expected array

**Comment by Kevin222004** [comment]
pls review

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/whitespacearound/InputWhitespaceAroundAllowEmptyTypesAndNonEmptyClasses2.java (hunk 1 above)
Comment (Kevin222004) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/whitespacearound/InputWhitespaceAroundAllowEmptyTypesAndNonEmptyClasses2.java]: done

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/whitespacearound/InputWhitespaceAroundAllowEmptyTypesAndNonEmptyClasses2.java (hunk 2 above)
Comment (Kevin222004) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/whitespacearound/InputWhitespaceAroundAllowEmptyTypesAndNonEmptyClasses2.java]: done

**Comment by Kevin222004** [comment]
@romani can you please guide why this (wreker/build - wrecker pipeline)  is failing. I have clicked in details and tried to understand but i cannot get anything

**Comment by Kevin222004** [comment]
@romani  please review

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/whitespacearound/InputWhitespaceAroundAllowEmptyTypesAndNonEmptyClasses2.java [hunk 3]
```diff
@@ -23,29 +23,31 @@
 import java.util.function.Function;
 import java.util.function.Supplier;
 
-public class InputWhitespaceAroundAllowEmptyTypesAndNonEmptyClasses2{ // violation
+    class EmptyAndNonEmptyClasses2{ int x; } // violation ''{' is not preceded with whitespace'
```
Comment (romani) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/whitespacearound/InputWhitespaceAroundAllowEmptyTypesAndNonEmptyClasses2.java:26]: move such new classes to the end of file, to keep top public class be top in file.
make indentation of EmptyAndNonEmptyClasses2 to start from beginning of file.
same for other changes line this.

**Review by romani — CHANGES_REQUESTED** [review]
item:

**Comment by Kevin222004** [comment]
@romani  Please review changes

**Review by romani — APPROVED** [review]
Ok to merge

**Pull request merged**